apps/end-tenancy/views/content/cookies.md

```diff
@@ -29,10 +33,11 @@ Google Analytics stores information about:
 
 No personal details are stored with this information, so you can’t be identified.
 
-| Name |    Purpose                    |  Expires   |
-|:-----|:--------------------------    |:---------  |
-| _ga  | Used to distinguish users     | 2 years    |
-| _gat | Used to throttle request rate | 10 minutes |
+| Name |    Purpose                         |  Expires   |
+|:-----|:-------------------------------    |:---------  |
+| _ga  | Used to distinguish users          | 2 years    |
+| _gat | Used to throttle request rate      | 10 minutes |
+| _gid | Used to count and track page views | 24 hours   |
 
 <div class="panel-indent">
   <p>Google isn’t allowed to use or share our analytics data.</p>
```

Comment:
In the PR it says changing the md because you're not using it, shouldn't you just delete it then?

Reply:
deleted cookies.md